### PR TITLE
Bug 1244593 - [Web Apps] There has no production Web Apps in app deck

### PR DESCRIPTION
**Pull request by josyeh**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=226bceb478bae66259ea1b44129d2d9c4960a542

File: build/config/tv/apps-engineering.list [hunk 1]
```diff
@@ -15,6 +15,7 @@ tv_apps/remote-control
 tv_apps/remote-control-client
 tv_apps/fling-tutorial
 tv_apps/tv-ftu
+tv_apps/webapps
```
Comment (evelynhung) [inline build/config/tv/apps-engineering.list]: Please also add to apps-production.list

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3f3aa8dd1ce2b631e9f1fb03b9101e8207729a79

File: build/config/tv/apps-engineering.list (hunk 1 above)
Comment (jostw) [inline build/config/tv/apps-engineering.list]: Okay, done.

**Pull request merged**